install.sh

```
@@ -0,0 +1,418 @@
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)